### DISCUSSION TITLE
Image Deletion Event for External Handling

### DISCUSSION THREAD

**Discussion by 0l1v3rr** [ueberdosis/tiptap#4801] [General Questions]
Hello everyone,

I'm currently working with the Tiptap editor and need some guidance on how to handle image deletion events.

I have a requirement to synchronize external storage with the images used in the editor. However, I haven't been able to find a clear way to detect when an image is deleted, making it difficult to maintain consistency between editor content and external resources.

Question:
Is there a recommended way to react when an image is deleted in the editor?

Thanks.

**Comment by IshfaqAhmedProg** [comment]
Have you found any solutions?
